test/test_other.py

```diff
@@ -3411,6 +3411,15 @@ def test_jspi_add_function(self):
       '-Wno-experimental']
     self.do_runf('other/test_jspi_add_function.c', 'done')
 
+  @requires_jspi
```

Comment:
Should we test this with both ASYNCIFY modes?

Reply:
Is there a decorator to parameterize over the asyncify modes? Or do I just do it manually>